### PR TITLE
Use CI version of Python

### LINKED ISSUE #154: Windows CI Matrix only tests Python 3.12
Despite how it looks in GitHub, the Winows GitHub Actions only test the nuget package that are in the Project file, which is hard-coded to 3.12.4.

### PR DESCRIPTION
**Pull request by RussKie**
Resolves #154

### CODE REVIEW

**Comment by RussKie** [comment]
The publish step is failing with auth errors:

image

File: src/Integration.Tests/Integration.Tests.csproj [hunk 1]
```diff
@@ -77,15 +22,18 @@
...
+  </ItemGroup>
 
-	<ItemGroup>
-		<Content Include="python\requirements.txt">
-		  <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-		</Content>
-		<Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
-	</ItemGroup>
+  <ItemGroup>
+    <AdditionalFiles Include="python\*.py">
```
Comment (tonybaloney) [inline src/Integration.Tests/Integration.Tests.csproj:31]: Oh right, this makes more sense!

**Review by tonybaloney — APPROVED** [review]
Nice work!

**Comment by tonybaloney** [comment]
> The publish step is failing with auth errors: image
> 
> Also, this: image

@aaronpowell

**Pull request merged**